src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -128,12 +127,16 @@ export default class FixedHeaderTable<T> extends React.Component<
         const tableDataStore = new FixedHeaderTableDataStore(() => {
             return this.props.data;
         }, this.props.fixedTopRowsData || []);
+        const filterString = toJS(this._store.filterString);
         this._store.setProps({
             columns: nextProps.columns,
             dataStore: tableDataStore,
             initialSortColumn: this._sortBy,
             initialSortDirection: this._sortDirection,
         });
+        if (filterString !== undefined) {
```

Comment:
interesting.  i think we should have comment explaining why we read and then set this again